server/lib/routes.js

```diff
@@ -110,7 +110,9 @@ module.exports = function (config, i18n) {
       '/account_unlock_complete',
       '/signup_permissions',
       '/signin_permissions',
-      '/unexpected_error'
+      '/unexpected_error',
+      '/openid',
+      '/start'
```

Comment:
Naming is hard, but here's a few thoughts:

* "/start" seems too generic to me, it doesn't really indicate anything about what that page is going to do.  Could it be e.g. "/openid/start" and "/openid/complete" or similar?.  Or maybe there's a more descriptive name that will give the impression of "choose how you want to sign in"?

* Will we need different endpoints for different variants of OpenID, e.g. the OpenID 2.0 stuff we already have versus some future partner that uses OpenID Connect?  If so, it may be worth baking that into the naming here somehow so we can grow sanely in future.